test/bin/find.js

```diff
@@ -108,4 +108,35 @@ describe('bin', function() {
     proxyquire('../../src/bin/find', stub)
     assert.ok(getCurrentRules.called)
   })
+
+  it('logs core rules', function() {
+    var stub = {
+      '../lib/rule-finder': function(specifiedFile, noCore) {
+        return {
+          getCurrentRules: function() {
+            assert(!noCore)
+            return ['current', 'rules']
+          }
+        }
+      },
+    }
+    process.argv[2] = '-c'
+    proxyquire('../../src/bin/find', stub)
+  })
+
+  it('does not log core rules', function() {
+    var stub = {
+      '../lib/rule-finder': function(specifiedFile, noCore) {
+        return {
+          getCurrentRules: function() {
+            assert(noCore)
+            return ['current', 'rules']
+          }
+        }
+      },
+    }
+    process.argv[2] = '-c'
```

Comment:
why use `-c` and then `--no-core`, just to demonstrate that it turns it off? perhaps a third test should be added that only does `--no-core`?